src/enumValues.ts

```diff
@@ -7,6 +7,11 @@ export class EnumValues {
     return Object.keys(e).filter(key => isNaN(+key))
   }
 
+  static getNameFromValue(e:any, l:(number|string)) {
+    var all= this.getNamesAndValues(e).filter(val=> { return val.value == l});
```

Comment:
change `var` to `const`
change `val=> { return val.value == l}` to `pair => pair.value === value`
also fix the spacing